gallery/src/main.scss

```diff
@@ -9,41 +9,63 @@
 @import '~@sonatype/react-shared-components/scss-shared/nx-container-helpers';
 @import '~@sonatype/react-shared-components/scss-shared/nx-flexbox-helpers';
 
+.nx-page-sidebar.gallery-page-sidebar {
+  padding-right: $nx-spacing-md;
+  padding-left: $nx-spacing-md;
+}
+
 .gallery-nav, .gallery-nav-links {
   @include container-vertical;
 }
 
-.gallery-nav-category {
-  @include semi-bold();
+.gallery-nav {
+  .nx-tree-view__trigger {
+    .nx-tree-view__text {
+      @include semi-bold();
 
-  color: $nx-text-color-light;
-  font-size: 16px;
-  margin-top: 16px;
-  margin-bottom: 12px;
+      color: #393939;
+      font-size: 16px;
+    }
+  }
 }
 
-.gallery-nav-link {
+.nx-tree-view__child.gallery-nav-link {
+  font-size: 16px;
+  height: 32px;
+  line-height: 32px;
+  margin-right: 0;
   margin-bottom: 4px;
   margin-left: 0;
 }
 
 .gallery-link {
+  border-radius: 6px;
+  box-sizing: border-box;
+  color: $nx-text-color-light;
   display: inline-block;
   font-size: 14px;
+  height: 32px;
   margin-bottom: 4px;
+  padding-left: 45px;
   text-decoration: none;
-  color: $nx-text-color-light;
+  width: 100%;
 
   &:hover {
-    text-decoration: underline;
+    background-color: #e2e6f4;
   }
 
   &.active {
-    @include semi-bold;
+    @include bold;
 
-    color: $nx-text-color;
+    background-color: #1a2eac;
+    color: #fff;
     cursor: default;
     text-decoration: none;
+
+    &:hover {
+      background-color: #1a2eac;
+      color: #fff;
+    }
```

Comment:
The amount of custom tree view styles here surprises me.  Is it because we don't have selectable items in the normal tree views at all?  If we did, would they follow these styles?

Reply:
> The amount of custom tree view styles here surprises me. Is it because we don't have selectable items in the normal tree views at all? If we did, would they follow these styles?

There are :hover states but no selected states. The second part is a question for @dturner32

Reply:
> There are :hover states but no selected states. The second part is a question for @dturner32

What about all the sizing changes?

Reply:
> I'm uncomfortable applying these changes to the nx-tree-view component directly until we have a full design document from @dturner32

So is restyling the tree views now a 1.0 task? Do we have a ticket for that?

Reply:
> > I'm uncomfortable applying these changes to the nx-tree-view component directly until we have a full design document from @dturner32
> 
> So is restyling the tree views now a 1.0 task? Do we have a ticket for that?

I have asked @dturner32 what he wants to do.

Reply:
> > > I'm uncomfortable applying these changes to the nx-tree-view component directly until we have a full design document from @dturner32
> > 
> > 
> > So is restyling the tree views now a 1.0 task? Do we have a ticket for that?
> 
> I have asked @dturner32 what he wants to do.

From @dturner32 "so the custom style for the side-bar was intended to be used only for that instance. As for the current treeview, those styles will remain the same"

Reply:
I'm ok with this since we're using the tree view explicitly for navigation which is a new use. I would like to see us develop this into general nav styles if we intend to use tree views in this way elsewhere